### PR TITLE
origin: Use odf stable-4.15 for baremetalds kv

### PR DESCRIPTION
**Pull request by qinqon**

### CODE REVIEW

**Comment by qinqon** [comment]
Capacity error 
```
100    57  100    57    0     0    270      0 --:--:-- --:--:-- --:--:--   270
Device info: f2488b8f-30be-4cf8-85ba-4c07eeee8512 null null
baremetalds: Failed to create equinix device: ipi-ci-op-388w8zp4-065bc-1725443869719400448
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   285  100     2  100   283     37   5240 --:--:-- --:--:-- --:--:--  5277
```
/pj-rehearse pull-ci-openshift-origin-master-e2e-baremetalds-kubevirt

File: ci-operator/config/openshift/origin/openshift-origin-master.yaml [hunk 1]
```diff
@@ -398,7 +398,7 @@ tests:
     cluster_profile: equinix-ocp-metal
     env:
       HYPERSHIFT_NODE_COUNT: "2"
-      ODF_OPERATOR_SUB_CHANNEL: stable-4.13
+      ODF_OPERATOR_SUB_CHANNEL: stable-4.15
```
Comment (maiqueb) [inline ci-operator/config/openshift/origin/openshift-origin-master.yaml:401]: I'm unsure (let me highlight unsure ...) if stable-4.15 exists for now. From what I see in openshift release, 4.15 is dev-preview for the time being.

Comment (qinqon) [reply ci-operator/config/openshift/origin/openshift-origin-master.yaml:401]: then what is the channel ? dev-preview-4.15 ?

**Pull request closed without merging**